test/test-v8debugapi.ts

```diff
@@ -110,6 +110,37 @@ function validateBreakpoint(breakpoint: stackdriver.Breakpoint): void {
     breakpoint.stackFrames.forEach(validateStackFrame);
   }
 }
+describe('v8debugapi', function() {
+  const config: DebugAgentConfig = extend({}, defaultConfig, {
+    workingDirectory: __dirname,
+    forceNewAgent_: true
+  });
+  const logger = new common.logger({ levelLevel: config.logLevel } as any as LoggerOptions);
+
+  it('Verify the correct debugapi was constructed', (done) => {
```

Comment:
Can this be moved in the existing `describe` block so we don't have an extra `describe` block if not needed.  If not, we should change the name of the `describe` block to be more specific.

Reply:
the other describe block is 'v8debugapi' but the debugapi is created in beforeEach(). I would be not convenient to create another debug in it block.